README.rst

```diff
@@ -19,6 +19,8 @@ Hopefully this is enough to get you going::
     -v, --verbose
     -d, --dump
     -u UUID, --uuid=UUID  Camera UUID
+    -i id, --id=id        Camera Connection id
+    -c, --connect_with_id Connect using id instead of UUID (NVR Beta API)
```

Comment:
Okay, I'm surprised, hadn't really seen a change in the login/snapshot API through multiple major revs of the software. I hope this isn't a sign of more breaking changes to come :/